.github/workflows/.github/workflows/webext-lint.yml

```
@@ -0,0 +1,23 @@
+name: Web-Ext lint
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+  workflow_dispatch:
+
+jobs:
+  lint:
+    name: "Lint"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: "web-ext lint"
+        uses: kewisch/action-web-ext@v1
```

```suggestion
        uses: kewisch/action-web-ext@cb8a694
```

Like this or what do you want here?